src/create-probot-app.ts

```diff
@@ -1,334 +1,25 @@
 #!/usr/bin/env node
-
-import fs from "fs-extra";
-import path from "path";
-import commander from "commander";
-import kebabCase from "lodash.kebabcase";
-import inquirer, { Answers, Question, QuestionCollection } from "inquirer";
-import validatePackageName from "validate-npm-package-name";
-import { guessEmail, guessAuthor, guessGitHubUsername } from "conjecture";
-import { generate } from "egad";
-
-import jsesc from "jsesc";
-import camelCase from "lodash.camelcase";
-import npm from "npm";
-import stringifyAuthor from "stringify-author";
-
+import { getAnswers, getProgram } from "./helpers/user-interaction";
 import { initGit } from "./helpers/init-git";
-import {
-  blue,
-  bold,
-  green,
-  printSuccess,
-  red,
-  writeHelp,
-  yellow,
-} from "./helpers/write-help";
-
-// TSC mangles output directory when using normal import methods for
-// package.json. See
-// https://github.com/Microsoft/TypeScript/issues/24715#issuecomment-542490675
-const pkg = require(require.resolve("../package.json"));
-
-/**
- * Partially sanitizes keys by escaping double-quotes.
- *
- * @param {Object} object The object to mutate.
- * @param {String[]} keys The keys on `object` to sanitize.
- */
-function sanitizeBy(
-  object: {
-    [key: string]: string;
-  },
-  keys: string[]
-): void {
-  keys.forEach((key) => {
-    if (key in object) {
-      object[key] = jsesc(object[key], {
-        minimal: true,
-        quotes: "double",
+import { installAndBuild } from "./helpers/run-npm";
+import { makeScaffolding } from "./helpers/filesystem";
+import { printSuccess, red } from "./helpers/write-help";
+
+function main(): void {
+  const program = getProgram();
+
+  getAnswers(program, program.destination).then((answers) => {
```

Comment:
Prefer `await` on `getAnswers` to *dedent* the following `.then((answers)...` block for easier readability.

```diff
diff --git a/src/create-probot-app.ts b/src/create-probot-app.ts
index 681894b..ed835f0 100644
--- a/src/create-probot-app.ts
+++ b/src/create-probot-app.ts
@@ -5,21 +5,19 @@ import { installAndBuild } from "./helpers/run-npm";
 import { makeScaffolding } from "./helpers/filesystem";
 import { printSuccess, red } from "./helpers/write-help";
 
-function main(): void {
+async function main(): Promise<void> {
   const program = getProgram();
-
-  getAnswers(program, program.destination).then((answers) => {
-    makeScaffolding(program.destination, answers, program.overwrite)
-      .then(() => {
-        if (program.gitInit) initGit(program.destination);
-      })
-      .then(() => installAndBuild(program.destination, answers.toBuild))
-      .then(() => printSuccess(answers.appName, program.destination))
-      .catch((err) => {
-        console.log(red(err));
-        process.exit(1);
-      });
-  });
+  const answers = await getAnswers(program, program.destination)
+  makeScaffolding(program.destination, answers, program.overwrite)
+    .then(async () => {
+      if (program.gitInit) await initGit(program.destination);
+    })
+    .then(() => installAndBuild(program.destination, answers.toBuild))
+    .then(() => printSuccess(answers.appName, program.destination))
+    .catch((err) => {
+      console.log(red(err));
+      process.exit(1);
+    });
 }

```

Reply:
Fine for me, feel free to send the patch already (I don't know yet how to send changes proposals directly in the PR but I saw you did before, I'll figure it out)